### DISCUSSION TITLE
Where to download map of Berlin

### DISCUSSION THREAD

**Discussion by EricGongjc** [mapsforge/mapsforge#1327] [Q&A] [answered]
Where to download berlin.map

**Comment by devemux86** [comment, marked as answer]
- [Mapsforge map providers](https://github.com/mapsforge/mapsforge/blob/master/docs/Mapsforge-Maps.md)

**Reply by EricGongjc** [reply]
thanks

**Reply by EricGongjc** [reply]
Do you have a map of China?

**Reply by devemux86** [reply]
Please search each map provider,
they have maps for all countries.

**Reply by EricGongjc** [reply]
OK，3Q，I'm going to find it

**Reply by EricGongjc** [reply]
thanks,I'm find it,If I need to change the map display, what should I change?

**Reply by EricGongjc** [reply]
i find a china.map,but i think to show hangzhou city,what should I change?

**Reply by EricGongjc** [reply]
@devemux86 help...

**Reply by EricGongjc** [reply]
I want to show Hangzhou city in android pad,but china.map is so big...

**Reply by EricGongjc** [reply]
Do you have a map of Chinese cities?

**Comment by devemux86** [comment]
Please create separate discussions to discuss different topics.

- Study the library documents (linked in the [main page](https://github.com/mapsforge/mapsforge)), for render themes, etc.
- Study the [mapsforge-samples-android](https://github.com/mapsforge/mapsforge/tree/master/mapsforge-samples-android) to see how to use the library in an app.
